### ISSUE TITLE
Errors because of undefined \LocallyStopLineNumbers and \ResumeLineNumbers

### ISSUE THREAD

**Issue by komascript** [Jinwen-XU/colorist#5]
A minimal example

```latex
\documentclass{colorbook}
\title{Title} 
\author{John Doe}
\begin{document}
\maketitle
\end{document}
```

results in two error messages, because of undefined control sequences:

```text
./test.tex:5: Undefined control sequence.
\maketitle ...itle/before}\LocallyStopLineNumbers 
                                                  \ProjLibTitlePage {,title=...l.5 \maketitle
              
(/usr/local/texlive/2022/texmf-dist/tex/latex/tex-gyre/t1qhv.fd) [1{/usr/local/texlive/2022/texmf-var/fonts/map/pdftex/updmap/pdftex.map}] [2]
./test.tex:5: Undefined control sequence.
\maketitle ...or ,date=\@date }\ResumeLineNumbers 
                                                  \UseHook {cmd/maketitle/af...l.5 \maketitle
```

I do not know, if this is intended. Al least this usually wouldn't be expected.

**Comment by Jinwen-XU** [comment]
Thank you for reporting this bug! It has been fixed with Update 2023-01-07 (845a4e95b965ab3658cc33b6c71ce7c730854f31).

This part of code is very old, and since I hardly write `\maketitle` in books, it was previous unnoticed.

P.S. Sorry that in the commit I mistakenly wrote 2022-01-07, still haven't gotten used to the new year :p

**Comment by komascript** [comment]
Thank you.
